src/richie/apps/courses/models/category.py

```diff
@@ -86,6 +86,7 @@ def get_category_limit_choices_to():
         "publisher_is_draft": True,  # plugins work with draft instances
         "category__isnull": False,  # limit to pages linked to a category object
         "node__parent__cms_pages__category__isnull": False,  # exclude meta categories
+        "node__parent__cms_pages__publisher_is_draft": True,  # exclude published categories
```

Comment:
```suggestion
        "node__parent__cms_pages__publisher_is_draft": True,  # # exclude category published parent
```

+ add the same paragraph as what you added for the course below?